osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -138,12 +138,15 @@ public void Stop()
         /// Resets this <see cref="GameplayClockContainer"/> and the source to an initial state ready for gameplay.
         /// </summary>
         /// <param name="time">The time to seek to on resetting. If <c>null</c>, the existing <see cref="StartTime"/> will be used.</param>
-        /// <param name="startClock">Whether to start the clock immediately, if not already started.</param>
+        /// <param name="startClock">Whether to start the clock immediately. If <c>false</c>, the clock will remain stopped after this call.</param>
```

Comment:
Okay maybe aside from this since it's not really true.
```suggestion
        /// <param name="startClock">Whether to start the clock immediately. If <c>false</c> and the clock was already paused, the clock will remain paused after this call.
        /// </param>
```

Reply:
Hmm.. I'm wondering if we can change that behaviour altogether then. Because with it it feels very wish-washy. I was hoping to make this xmldoc imply that this call will stop the clock and get it into a good state.

Reply:
I'd much prefer to make such a change in a separate pull if I'm honest with you, because it makes the change 300% scary. It brings it from "oh this is a quaint little obvious fix" to "I have to check every single gameplay clock usage for potentially relying on the reset-while-running behaviour lest it gets stuck forevermore".

Reply:
Sure thing.